.gitignore

```diff
@@ -92,10 +92,6 @@ system-tests/lib/fixtureDirs.ts
 /packages/frontend-shared/src/generated
 /packages/frontend-shared/cypress/e2e/support/e2eProjectDirs.ts
 
-# from npm/create-cypress-tests
-/npm/create-cypress-tests/initial-template
-/npm/create-cypress-tests/src/test-output
```

Comment:
might be worth keeping this around so stale files don't accidentally get checked in when people change branches?

Reply:
https://github.com/cypress-io/cypress/pull/28472/commits/2b158f5b838f946198cbd7ec3724b27e198dd8bd